### ISSUE TITLE
[FEATURE] Release Python wheels for 3.9 and 3.11

### ISSUE THREAD

**Issue by ralbertazzi** [CVCUDA/CV-CUDA#91]
Duplicate of https://github.com/CVCUDA/CV-CUDA/issues/78 that was closed with

> we plan on supporting both interpreters in the next release.

The next release has been.. released but still without support for Python 3.9 and 3.11. I'm re-opening the issue so that you can keep track of it.

Could you please close the issue only once wheels for Python 3.9 and 3.11 have been released? 🙏

**Comment by dlesage-nvidia** [comment]
Hi, thank you for your interest in CV-CUDA! 
The most recent release (v0.5, Beta 3) comes with Python debian packages for Python 3.8, 3.9, 3.10, 3.11 and wheels for Python 3.8, 3.9 and 3.10.
I will follow-up on the wheels for Python 3.11.

Thank you,

**Comment by bhaefnerNV** [comment]
Hi @ralbertazzi,
Thank you for your interest in CVCUDA.
Since [release 0.6.0-beta](https://github.com/CVCUDA/CV-CUDA/releases/tag/v0.6.0-beta), we provide standalone python wheels, including tooling and documentation to generate them for python versions 3.8, 3.9, 3.10, and 3.11. The python wheels can be under the same link above. If you'd to build your own python wheels, this can automatically be triggered, when building CVCUDA in release mode, please see [here ](https://github.com/CVCUDA/CV-CUDA#build-from-source)for further details.
We will close this issue for now. Please feel free to reopen, if more questions arise.
